wordpress-mobile/WordPress-iOS
# Fix constraints breaking during presentation

Similar to table view cell autosizing, the height of the container view may not always be _exactly_ correct during presentation. Because of this, constraints should be flexible and allow breakage temporarily in these scenarios.

Part of #13320

To test:
- Tap the Create Button to display the modally presented action sheet (not in a popover).
- Before the change, a constraint warning should appear, breaking the `sitePageButton` height constraint.
- After the change, no warning should appear.

PR submission checklist:

- [x] ~I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.~ Unreleased
